windows_bat/PokemonGo-Bot-Install.bat

```diff
@@ -203,14 +201,9 @@ if exist %PGBotPath%\encrypt. copy %PGBotPath%\encrypt. %DownPath%
 if exist %PGBotPath% rmdir %PGBotPath% /s /q
 if not exist %PGBotPath% md %PGBotPath%
 cd C:\Python27\
-pip2 install --upgrade virtualenv

```

Comment:
I removed because after testing in some Windows machines it would cause issues in some of them. Pip just wouldnt be able to get setuptools from the online repositories, so it stuck on a loop and the bot would never start. Removing virtualenv made the bot work, but makes the user unable to run multiple bots simultaneously. I think this is fine, as these batches are intended to be used by people with very low knowledge of how to configure the bot, and running multiple bots require a bit more understanding of how it Works.